01OS/01OS/clients/base_device.py

```diff
@@ -95,6 +158,8 @@ def record_audio(self):
                 send_queue.put({"role": "user", "type": "audio", "format": "audio/wav", "content": ""})
                 send_queue.put({"role": "user", "type": "audio", "format": "audio/wav", "end": True})
         else:
+            self.queue_all_captured_images()
```

Comment:
@KillianLucas When I send the images to the backend, they show up in the conversation.json, but Open Interpreter doesn't seem to be doing anything with them. I'm a little confused about how to implement this on the server side. Do you have any advice?

Reply:
That makes sense, the model in OI needs to support vision, we by default use GPT-3.5 which does not. 

Once we have some logic for taking an image (path or base64), sending it to an API (or ideally, running it through Moondream locally) and receiving a description, we can use that to "render" images to text only LLMs.

If you wanted to stand that pipeline up in an independent python file, I could add it as a tool to the 01 (so it can describe images, given a filepath) and to OI (so images are automatically rendered to text-only models). If not, I'll add it as a ticket for the 01!